app/controllers/provider_interface/conditions_controller.rb

```diff
@@ -40,5 +41,9 @@ def update
 
       redirect_to provider_interface_application_choice_path(@application_choice.id)
     end
+
+    def redirect_back_if_application_recruited
+      redirect_back(fallback_location: provider_interface_path) if @application_choice.recruited?
```

Comment:
Is it safe to use `TERMINAL_STATES` instead to guard against all those or should it only be the recruited state?
Also I was wondering if we should use `provider_interface_application_choice_path(@application_choice.id)` here instead rather than `/provider`?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4278/commits/c54844635a3fc8794e4ca1ad2166e6a17d65be34